defaults/main.yml

```diff
@@ -9,4 +9,6 @@ etcd_ca_depot: ""
 etcd_certs_root: "/etc/ssl/etcd"
 local_certs: "certs"
 
-etcd_core_group: ""
\ No newline at end of file
+etcd_core_group: ""
+
+etcd_image_tag: "v3.1.0"
```

Comment:
I meant the location of etcd image. If it is pulled from coreos the setup might have issues like we have with existing images taking a lot of time to download.